yt_dlp/extractor/common.py

```diff
@@ -789,7 +789,8 @@ def _webpage_read_content(self, urlh, url_or_request, video_id, note=None, errno
             self._downloader.to_screen(dump)
         if self.get_param('write_pages', False):
             basen = '%s_%s' % (video_id, urlh.geturl())
-            if len(basen) > 240:
+            trim_length = self._downloader.params.get('trim_file_name', False) or 240
```

Comment:
This is how elsewhere is handled, so I added it.

Reply:
```suggestion
            trim_length = self.get_param('trim_file_name') or 240
```